packages/transformers/prerender/src/PrerenderTransformer.js

```diff
@@ -0,0 +1,72 @@
+import {promises as fs} from 'fs';
+import path from 'path';
+import Prerenderer from '@prerenderer/prerenderer';
+import Puppeteer from '@prerenderer/renderer-puppeteer';
+import {Transformer} from '@parcel/plugin';
+
+export default (new Transformer({
```

Comment:
This transformer should run when everything is bundled. Not sure how to force that using the current API. The output of this transformer should then be fed to htmlnano, posthtml, etc.

Reply:
> This transformer should run when everything is bundled 

It has to be an optimizer (or packager) for that, this would make it run before the existing htmlnano optimizer:

```json
{
	"extends": "@parcel/config-default",
	"optimizers": { "*.html": ["your-thing", "..."] }
}
```

Reply:
The optimizer doesn't support returning multiple assets. How can we use that?